build.sbt

```diff
@@ -5,7 +5,7 @@ name := "soy"
 organization := "com.kinja"
 
 // We use Semantic Versioning. See: http://semver.org/
-version := "0.4.4" + {if (System.getProperty("JENKINS_BUILD") == null) "-SNAPSHOT" else ""}
+version := "0.5" + {if (System.getProperty("JENKINS_BUILD") == null) "-SNAPSHOT" else ""}
```

Comment:
Now that I think about it, shouldn't you change the version to `1.0.0` since this change ia not backward compatible?